package.json

```diff
@@ -92,8 +92,9 @@
         "@powerhousedao/design-system": "1.0.0-alpha.75",
         "did-key-creator": "^1.2.0",
         "document-drive": "1.0.0-alpha.17",
-        "document-model": "1.0.32",
-        "electron-is-dev": "^2.0.0",
+        "document-model": "^1.0.33",
+        "electron-deeplink": "^1.0.10",
```

Comment:
I'm not sure, but from what I remember, this lib was causing some errors when we try to build the electron app. Can you validate that this version works well with the build please?

Reply:
It worked on my machine (osx), was it on other OS?